LICENSE

```diff
@@ -1,21 +0,0 @@
-MIT License
```

Comment:
MIT LICENSE IS GARBAGE PLEASE SWITCH TO GPL THANKS

Reply:
SOMETHING THAT DOESNT EXIST DOESNT NEED GPL BUDDY NO LICENSE REQUIRED FOR THIS TRASH FEEL FREE TO USE AND ABUSE

Reply:
EXACTLY